src/posix/platform/system.cpp

```diff
@@ -66,10 +66,51 @@ static void processStateChange(otChangedFlags aFlags, void *aContext)
 }
 #endif
 
+static const char *get802154RadioUrl(otPlatformConfig *aPlatformConfig)
+{
+    const char *radioUrl = nullptr;
+
+    for (uint8_t i = 0; i < aPlatformConfig->mRadioUrlNum; i++)
+    {
+        ot::Posix::RadioUrl url(aPlatformConfig->mRadioUrls[i]);
+        if (strcmp(url.GetProtocol(), "trel") == 0)
+        {
+            continue;
+        }
+        else
+        {
+            radioUrl = aPlatformConfig->mRadioUrls[i];
+            break;
+        }
+    }
+
+    VerifyOrDie(radioUrl != nullptr, OT_EXIT_INVALID_ARGUMENTS);
+    return radioUrl;
+}
+
+#if OPENTHREAD_CONFIG_RADIO_LINK_TREL_ENABLE
+static const char *getTrelRadioUrl(otPlatformConfig *aPlatformConfig)
+{
+    const char *radioUrl = nullptr;
+
+    for (uint8_t i = 0; i < aPlatformConfig->mRadioUrlNum; i++)
+    {
+        ot::Posix::RadioUrl url(aPlatformConfig->mRadioUrls[i]);
+        if (strcmp(url.GetProtocol(), "trel") == 0)
```

Comment:
nit: add empty line after declaration
```suggestion
        ot::Posix::RadioUrl url(aPlatformConfig->mRadioUrls[i]);

        if (strcmp(url.GetProtocol(), "trel") == 0)
```